engine/crash/wscript

```diff
@@ -53,6 +53,9 @@ def configure(conf):
     if operating_sys == "win":
         conf.env.append_value('LINKFLAGS', ['dbghelp.lib'])
 
+    conf.env.append_value('CFLAGS', ["-fdebug-compilation-dir=engine/crash", "-fdebug-prefix-map=../src=src", "-fdebug-prefix-map=../../../tmp/dynamo_home=../../defoldsdk"])
+    conf.env.append_value('CXXFLAGS', ["-fdebug-compilation-dir=engine/crash", "-fdebug-prefix-map=../src=src", "-fdebug-prefix-map=../../../tmp/dynamo_home=../../defoldsdk"])
+
```

Comment:
These should be set in waf_dynamo.py, where we set common build settings.

Also, how will this work for windows, where we currently use cl.exe?

Reply:
`"-fdebug-compilation-dir="` has different values for different modules. `-fdebug-prefix-map=` I will move to common flags.

> Also, how will this work for windows, where we currently use cl.exe?
I understood in the way that on Windows we also use clang to build. Will recheck

Reply:
Ah, I missed the initial commonalities.

Still, I assume it will be a lot of errors or warnings for windows and xbox that uses cl.exe

Reply:
I wrapped flags with check for target OS because for Windows there are no direct analog for `-fdebug-*` flags.